lighthouse-cli/index.js

```diff
@@ -62,6 +62,7 @@ const cli = yargs
     'list-all-audits': 'Prints a list of all available audits and exits',
     'list-trace-categories': 'Prints a list of all required trace categories and exits',
     'config-path': 'The path to the config JSON.',
+    'launch-chrome': 'Launch chrome automatically when accessing port 9222 fails',
```

Comment:
I don't think it's possible. The CLI `--help` will say that it's a boolean and default is true, and to disable it `--no` should be pretty straight forward IMO even if its not obvious.